bindings/rust/src/lib.rs

```diff
@@ -162,18 +178,13 @@ impl KZGProof {
     }
 
     pub fn compute_kzg_proof(
-        blob: &[Blob],
```

Comment:
Just to note, this was a problem before. It is supposed to take a single blob. Testing helps.

Reply:
Also, just to be clear, this was my fault. I added this function a while ago.